<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Vec3Tuner</class>
 <widget class="QWidget" name="Vec3Tuner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QDoubleSpinBox" name="tuner_x"/>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="tuner_y"/>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="tuner_z"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
